admin/lib/solidus_admin/engine.rb

```diff
@@ -69,5 +69,11 @@ class Engine < ::Rails::Engine
         SolidusAdmin::BaseComponent.include app.routes.mounted_helpers
       end
     end
+
+    initializer "solidus_admin.load_ext" do
+      Dir[root.join("lib/solidus_admin/ext/**/*.rb")].sort.each do |file|
```

Comment:
This seems to be another way of loading concerns/monkeypatches. I am not sure we need that if we make use of Rails autoloading